server/internal/usecase/interfaces/common.go

```diff
@@ -12,9 +12,11 @@ const (
 
 var (
 	ErrOperationDenied error = errors.New("operation denied")
+	ErrFileNotIncluded error = errors.New("file not included")
```

Comment:
```suggestion
```
define it at `interfaces/asset.go` instead

Reply:
Done!